testplan/testing/cpp/cppunit.py

```diff
@@ -208,7 +210,7 @@ def test_command_filter(self, testsuite_pattern, testcase_pattern):
             )
 
         if testcase_pattern not in ("*", self._VERIFICATION_TESTCASE_NAME):
-            self.logger.debug(
+            self.logger.info(
```

Comment:
Wondering if this should be user_info due to the fact it is not about what is being executed one by one, but more of a summary statement that "look, we are going to run a whole test suite now". I will let others opine too.